src/frontend/components/SøknadsSteg/Oppsummering/Oppsummering.tsx

```diff
@@ -63,25 +85,220 @@ const Oppsummering: React.FC = () => {
+                        søknadsvar={landkodeTilSpråk(
+                            søknad.søker.arbeidsland.svar,
+                            intl.defaultLocale
+                        )}
+                    />
+                )}
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.utenlandspensjon.spm'} />}
+                    søknadsvar={søknad.søker.mottarUtenlandspensjon.svar}
+                />
+                {søknad.søker.pensjonsland.svar && (
+                    <OppsummeringFelt
+                        tittel={<SpråkTekst id={'omdeg.utenlandspensjon.land.spm'} />}
+                        søknadsvar={landkodeTilSpråk(
+                            søknad.søker.pensjonsland.svar,
+                            intl.defaultLocale
+                        )}
+                    />
+                )}
+            </Oppsummeringsbolk>
+
+            <Oppsummeringsbolk tittel={'hvilkebarn.sidetittel'}>
+                {søknad.barnInkludertISøknaden.map(barn => (
+                    <>
+                        {barn.navn && <OppsummeringFelt tittel={<SpråkTekst id={barn.navn} />} />}
+                        {barn.ident && (
+                            <OppsummeringFelt
+                                tittel={<SpråkTekst id={'hvilkebarn.barn.fødselsnummer'} />}
+                                søknadsvar={barn.ident}
+                            />
+                        )}
+
+                        {barn.borMedSøker ? (
+                            <OppsummeringFelt
+                                tittel={<SpråkTekst id={'hvilkebarn.barn.bosted'} />}
+                                søknadsvar={intl.formatMessage({
+                                    id: 'hvilkebarn.barn.bosted.din-adresse',
+                                })}
+                            />
+                        ) : (
+                            <OppsummeringFelt
+                                tittel={<SpråkTekst id={'hvilkebarn.barn.bosted'} />}
+                                søknadsvar={intl.formatMessage({
+                                    id: 'hvilkebarn.barn.bosted.ikke-din-adresse',
+                                })}
+                            />
+                        )}
+                    </>
+                ))}
+            </Oppsummeringsbolk>
+            <Oppsummeringsbolk tittel={'ombarna.sidetittel'}>
+                {søknad.erNoenAvBarnaFosterbarn !== undefined && (
+                    <OppsummeringFelt
+                        tittel={<SpråkTekst id={'ombarna.fosterbarn.spm'} />}
+                        søknadsvar={søknad.erNoenAvBarnaFosterbarn.svar}
+                    />
+                )}
+                {søknad.barnInkludertISøknaden.map(barn => (
+                    <>
+                        {barn.erFosterbarn.svar === 'JA' && (
+                            <OppsummeringFelt
+                                tittel={<SpråkTekst id={'ombarna.fosterbarn.hvem.spm'} />}
```

Comment:
Her tror jeg den vil rendre en overskrift for hvert barn?
Hva med å sjekke om `søknad.erNoenAvBarnaFosterbarn.svar === ESvar.JA` og hvis den er true, rendre `<OppsummeringsFelt` der søknadssvaret er en filtrert liste med barn?